app/locales/sk.json

```diff
@@ -0,0 +1,141 @@
+  "I've noted down the link.": "Zapísal som si tento odkaz.",
+  "Proceed": "Pokračovať",
+  "The link has been copied": "Odkaz bol skopírovaný.",
+  "Could not copy the link, please copy it yourself": "Odkaz sa nedá skopírovať, skopírujte si ho manuálne",
+  "Contributors": "Prispievatelia",
+  "This website is made with help from volunteers. In no particular order:": "Táto stránka je vytvorená s pomocou dobrovoľných prispievateľov. Bez konkrétneho poradia:"
+  "...and others": "...a ďalší",
+  "The code for the website can be found": "Zdrojový kód stránky nájdete na",
+  "here": "tu",
+  "About the service": "O službe",
+  "In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.": "V súvislosti s prepuknutím pandémie COVID-19 bolo na vírus testovaných veľa ľudí, ale z dôvodu nedostatku zdrojov sa drvivá väčšina netestovala. Preto sa predpokladá, že v súvislosti s ohniskom nákazy existuje veľké množstvo chýbajúcich údajov.",
+  "Using self-report, you can get a better picture of how many people have symptoms without exposing health workers to potential contamination hazards and without using up valuable infection control equipment that is already in short supply. We at Bustbyte, well helped by other volunteers, have created this tool in response to ": "Zaslaním vlastného reportu môžeme všetci získať lepšiu predstavu o tom, koľko ľudí má príznaky bez toho, aby boli zdravotnícki pracovníci vystavení potenciálnym nebezpečenstvám kontaminácie a bez spotrebovania cenného vybavenia na kontrolu infekcie, ktoré je nedostatkové. My v Bustbyte s pomocou iných dobrovoľníkov, sme vytvorili tento nástroj ako reakciu na tento článok",
+  "this article": "tento článok",
+  "as an attempt to chart these numbers.": "ako pokus vykresliť tieto čísla.",
+  "This is not an official tool of the authorities, and for more information about the outbreak we refer to health authorities which always have updated information.": "Toto nie je oficiálny web slovenských štátnych orgánov. Pre viac informácií o ohnisku nákazy Vám odporúčame navštíviť štátom spravované stránky, ktoré poskytujú aktualizované informácie.",
+  "https://who.org": "https://www.uvzsr.sk",
+  "To WHO.org": "Na uvzsr.sk",
+  "Developed by": "Vytvorené spol.",
+  "BustByte AS": "BustByte AS",
+  "and": "a",
+  "volunteers": "dobrovoľníkmi",
+  "Privacy Policy": "Ochrana osobných údajov",
+  "Privacy": "Súkromie",
+  "Thank you for your contribution!": "Ďakujeme za Váš príspevok!",
+  "The government has now launched their own self-report system, please use that instead.": "Štát už zriadil vlastný systém na samo ohlasovane stavu, prosím použite radšej tento."
```

Comment:
Missing comma

Reply:
thank you, will push in a batch fixed together with others